openforcefield/typing/engines/smirnoff/forcefield.py

```diff
@@ -507,20 +503,23 @@ def render_atoms(atomsets):
             raise Exception(
                 msg)  # TODO: Should we raise a more specific exception here?
 
-    def get_handler(self, tagname, handler_kwargs=None):
+    def get_parameter_handler(self, tagname, handler_kwargs=None, permit_cosmetic_attributes=False):
         """Retrieve the parameter handlers associated with the provided tagname.
 
-        If the parameter handler has not yet been instantiated, it will be created.
+        If the parameter handler has not yet been instantiated, it will be created and returned.
         If a parameter handler object already exists, it will be checked for compatibility
-        and an Exception raised if it is incompatible with the provided kwargs.
+        and an Exception raised if it is incompatible with the provided kwargs. If compatible, the
+        existing ParameterHandler will be returned.
 
         Parameters
         ----------
-        tagame : str
+        tagname : str
             The name of the parameter to be handled.
-        handler_kwargs : dict, optional. Default=None
+        handler_kwargs : dict, optional. Default = None
             Dict to be passed to the handler for construction or checking compatibility. If None, will be assumed
             to represent handler defaults.
+        permit_cosmetic_attributes : bool, optional. Default = False
```

Comment:
This is outrageously nitpicky, but we used `allow_` instead of `permit_` somewhere else (at least) in `allow_undefined_stereo`, and I wonder if we should change one of the two for naming consistency.

Reply:
Good point. I agree we should standardize on one, and am basically ambivalent about which. If I had to choose, I think `allow` is slightly clearer.